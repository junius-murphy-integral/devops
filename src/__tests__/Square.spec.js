import React from "react";
import { create } from "react-test-renderer";
import Square from "../Square";

describe("Square component", () => {
  test("Matches the snapshot", () => {
    const square = create(<Square />);
    expect(square.toJSON()).toMatchSnapshot();
  });
});