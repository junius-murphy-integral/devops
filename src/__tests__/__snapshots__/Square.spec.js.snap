// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Square component Matches the snapshot 1`] = `
<button
  className="square"
  onClick={[Function]}
/>
`;
